.github/workflows/c-cpp.yml

```diff
@@ -26,4 +26,4 @@ jobs:
     - name: Lint
       run: ./scripts/lint.sh
     - name: Run Tests
-      run: ./cmake-build-debug/hyriseTest
```

Comment:
According to our workflow this should work. See also comment in the ./scripts/format.sh.